src/test/skript/tests/syntaxes/effects/EffWorldBorderExpand.sk

```diff
@@ -0,0 +1,33 @@
+test "worldborder expand":
```

Comment:
For all tests, need more edge cases with infinite values, NaN values, negative values, etc
try your hardest to break your code